src/Form/AdminSettingsForm.php

```diff
@@ -206,9 +227,13 @@ private function buildFieldMappingsForm() {
     $field_mappings_settings = $this->config('acquia_lift.settings')->get('field_mappings');
     $field_names = $this->getTaxonomyTermFieldNames();
 
+    $taxonomy_vocabulary_url_text = t('Taxonomy Vocabulary');
+    $taxonomy_vocabulary_url = Url::fromRoute('entity.taxonomy_vocabulary.collection', [], ['attributes' => ['target' => '_blank']]);
+    $taxonomy_vocabulary_link = Link::fromTextAndUrl($taxonomy_vocabulary_url_text, $taxonomy_vocabulary_url)->toString();
+
     $form = [
       '#title' => t('Field Mappings'),
-      '#description' => t('Map Taxonomy vocabulary fields to "content section", "content keywords", and "persona" fields.'),
+      '#description' => t('Create ') . $taxonomy_vocabulary_link . t(' and map to "content section", "content keywords", and "persona" fields.'),
```

Comment:
I tried again and found what went wrong last time. Fixed in 8cca7b84d5d1f51308b51c89dd28b821fc0b9170.